src/main/scala/scalaz/distributed/package.scala

```diff
@@ -0,0 +1,7 @@
+package scalaz
+
+import scalaz.zio.IO
+
+package object distributed {
+  type F[A] = IO[Error, A]
```

Comment:
I'd name this more helpfully, maybe `Distributed[A] = IO[Error, A]`

Reply:
Done.